credentials/src/test/kotlin/web5/sdk/credentials/VerifiableCredentialTest.kt

```diff
@@ -167,25 +168,53 @@ class VerifiableCredentialTest {
     assertEquals(parsedVc.evidence, evidence)
   }
 
-  data class KnowYourCustomerCred(val country: String)
-  @Test
-  fun `kyc credential verify does not throw an exception if vc is legit`() {
-    val keyManager = InMemoryKeyManager()
-    val issuerDid = DidJwk.create(keyManager)
-    val subjectDid = DidJwk.create(keyManager)
+  data class KnownCustomerCredential(val id: String, val country_of_residence: String, val tier: String)
 
-    val vc = VerifiableCredential.create(
-      type = "KnowYourCustomerCred",
-      issuer = issuerDid.uri,
-      subject = subjectDid.uri,
-      expirationDate = Calendar.Builder().setDate(2055, 11, 21).build().time,
-      data = KnowYourCustomerCred(country = "us")
-    )
+@Test
+fun `kyc credential verify does not throw an exception if vc is legit`() {
+  val keyManager = InMemoryKeyManager()
+  val issuerDid = DidJwk.create(keyManager)
+  val subjectDid = DidJwk.create(keyManager)
 
-    val vcJwt = vc.sign(issuerDid)
-    VerifiableCredential.verify(vcJwt)
+  val expirationCalendar = Calendar.getInstance().apply {
+    set(2055, Calendar.DECEMBER, 21) // Note: Calendar months are zero-based in Java/Kotlin
   }
 
+  val expectedEvidence = listOf(
+    mapOf("kind" to "document_verification", "checks" to listOf("passport", "utility_bill")),
+    mapOf("kind" to "sanctions_check", "checks" to listOf("daily"))
+  )
+
+
+  val vc = VerifiableCredential.create(
+    type = "KnowYourCustomerCred",
+    issuer = issuerDid.uri,
+    subject = subjectDid.uri,
+    issuanceDate = Calendar.getInstance().time, // For the current time
+    expirationDate = expirationCalendar.time,
+    data = KnownCustomerCredential(id = subjectDid.uri, country_of_residence = "US", tier = "Tier 1"),
+    credentialSchema = CredentialSchema(
+      id = "https://schema.org/PFI",
```

Comment:
we should use a real schema here and update the verification logic to resolve and verify the json schema against the credential